save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/AwesomeBenchmarksView.kt

```diff
@@ -492,6 +508,19 @@ class AwesomeBenchmarksView : AbstractView<PropsWithChildren, AwesomeBenchmarksS
         }
     }
 
+    private fun changeUrl(selectedMenu: BenchmarkCategoryEnum?) {
```

Comment:
This method looks the same as methods in `ContestView`, `OrganizationView` and `ProejctView`. Maybe need to extract it to somewhere?

Reply:
Merged the methods into one